catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/DumpCommandTest.java

```diff
@@ -203,13 +209,54 @@ public void testNoFileDumpedWhenTransformFails() throws Exception {
     assertThat(consoleOutput.getOutput(), containsString(" 0 file(s) dumped in "));
   }
 
+  @Test
+  public void testNoZipFileDumpedWhenTransformFails() throws Exception {
+    // mock transformer to throw exception
+    MetacardTransformer transformer = mock(MetacardTransformer.class);
+    when(transformer.transform(any(), any())).thenThrow(CatalogTransformerException.class);
+    QueryResponseTransformer zipCompressionMock = mock(QueryResponseTransformer.class);
+    when(zipCompressionMock.transform(any(), any())).thenThrow(CatalogTransformerException.class);
+    List<MetacardTransformer> transformers = new ArrayList<>();
+    transformers.add(transformer);
+
+    // given
+    List<Result> resultList = getResultList("id1", "id2");
+    MetacardImpl metacard1 = new MetacardImpl(resultList.get(0).getMetacard());
+    MetacardImpl metacard2 = new MetacardImpl(resultList.get(1).getMetacard());
+    metacard1.setResourceURI(new URI("content:" + metacard1.getId()));
+    metacard2.setResourceURI(new URI("content:" + metacard2.getId() + "#preview"));
+
+    TestDumpCommand dumpCommand = new TestDumpCommand(transformers, zipCompressionMock);
+    dumpCommand.catalogFramework = givenCatalogFramework(resultList);
+    dumpCommand.filterBuilder = new GeotoolsFilterBuilder();
+    File outputDirectory = testFolder.newFolder("somedirectory");
+    String outputDirectoryPath = outputDirectory.getAbsolutePath();
+    dumpCommand.dirPath = outputDirectoryPath;
+    dumpCommand.transformerId = "someOtherTransformer";
+    Path zipfilePath = Paths.get(outputDirectoryPath.toString(), "foo.zip");
```

Comment:
✏️ `outputDirectoryPath` is already a String

Reply:
Thought it was an Path object. Fixed.